starwars/src/components/Character.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import styled from "styled-components";
+
+function Character(props) {
+  const {
+    name,
+    birth_year,
+    gender,
+    skin_color,
+    eye_color,
+    hair_color,
+    height,
+    mass
+  } = props;
+  console.log("character props", props);
+  return (
+    <div>
```

Comment:
This `div` wrapper is not needed anymore in this case because `CharacterTab` holds all other elements inside of it; it becomes a wrapper.